llmfoundry/models/layers/attention.py

```diff
@@ -34,6 +33,13 @@ def is_flash_v1_installed():
     return version.parse(flash_attn.__version__) < version.parse('2.0.0')
 
 
+if is_flash_v1_installed():
+    import transformers
+    transformers.utils.is_flash_attn_available = lambda: False
```

Comment:
@dakinggg dont you have a util for checking if flash v1 vs flashv2 is installed?